packages/vue/src/components/atoms/SfInput/SfInput.vue

```diff
@@ -23,7 +24,7 @@
       <span class="sf-input__bar"></span>
       <label
         :class="{ 'display-none': !label }"
-        class="sf-input__label"
+        class="sf-input__label sf-label"
```

Comment:
What do you think about changing the name of this class to sth that informs about what it is used for, like`will-change`?

Reply:
name changed